File: 프로그래머스/0/120844. 배열 회전시키기/배열 회전시키기.java
class Solution {
    public int[] solution(int[] numbers, String direction) {
        int[] answer = new int[numbers.length];
        int length = numbers.length;
        
        if (direction.equals("right")) {
            for (int i = 0; i < length - 1; i++) {
                answer[i + 1] = numbers[i];
            }
            answer[0] = numbers[length - 1];
        } else {
            for (int i = 0; i < length - 1; i++) {
                answer[i] = numbers[i + 1];
            }
            answer[length - 1] = numbers[0];
        }
        
        return answer;
    }
}

File: 프로그래머스/0/120844. 배열 회전시키기/배열 회전시키기.py
def solution(numbers, direction):
    answer = [0] * len(numbers)
    
    if direction == 'right':
        for i, num in enumerate(numbers):
            answer[(i + 1) % len(numbers)] = num
        return answer
    
    for i, num in enumerate(numbers):
        if i - 1 < 0:
            answer[len(numbers) - 1] = num
            continue
        
        answer[i - 1] = num
    return answer